maquetado/setup/scss/tech.scss

```diff
@@ -18,13 +18,14 @@
     bottom: 5px;
     z-index: -1;
     border-radius: 10px;
-    width: 190px;  
-    height: 250px;
+    width: 210px;  
+    height: 300px;
   }
 }
 
 .tech-img {
   height: 100px;
+  width: 100px;
```

Comment:
If the width is your limitting att, try removing the height. If it doesnt show the way you want that way, you can dismiss this comment

Reply:
My limiting attributes are both, therefore i used them